### ISSUE TITLE
Unable to deploy the repo

### ISSUE THREAD

**Issue by JoeHO888** [enuchi/React-Google-Apps-Script#26]
I follow the instruction in ReadMe, everything is working fine before I deploy the repo
`npm run deploy`

However, the script does not updated and google sheet does not have add-in installed, but the command line interface shows that I have already pushed files to google script project.
![file-uploaded](https://user-images.githubusercontent.com/29224653/76142423-8a57bf00-60a8-11ea-867a-188926ca3b13.png)
![EmptyScript](https://user-images.githubusercontent.com/29224653/76142428-9cd1f880-60a8-11ea-9ef1-5bb39bcf4c4a.png)

**Comment by enuchi** [comment]
Only the files in your `dist` folder should be pushed, but it looks like your whole project of 23000 files including `node_modules` is being pushed. Does your `.clasp.json` file in your project root look like this?
```
{
    "rootDir": "dist",
    "scriptId": "...paste scriptId here..."
}
```

**Comment by JoeHO888** [comment]
I solved the issue. Existence of ".clasp.json" in repo will make `npm run setup` fail, so I removed it. After bound to the sheets, I need to add "roodDir" Key Value Pair in ".clasp.json"

![error](https://user-images.githubusercontent.com/29224653/76154114-ca9f5780-6111-11ea-92b3-434cfd700007.png)

**Comment by enuchi** [comment]
Thanks for pointing this out @JoeHO888. This should be solved by #33.

(Running `npm run setup` now simply removes the `.clasp.json` file before running `clasp create`.)
